.github/workflows/typescript-generate.yml

```
@@ -0,0 +1,73 @@
+name: Generate typescript types
+on:
+  push:
+    branches:
+      - master
+      - typescript-generate
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.CI_AWS_CLIENT_ID }}
+  AWS_DEFAULT_REGION: "eu-west-1"
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.CI_AWS_CLIENT_SECRET }}
+  NDLA_AWS_ECR_REPO: ${{ secrets.NDLA_AWS_ECR_REPO }}
+  CI_RELEASE_ROLE: ${{ secrets.CI_RELEASE_ROLE }}
+  CI_GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+  DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+  DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
+  NDLA_RELEASES: ${{ secrets.NDLA_RELEASES }}
+  PACT_BROKER_PASSWORD: ${{ secrets.PACT_BROKER_PASSWORD }}
+  PACT_BROKER_URL: ${{ secrets.PACT_BROKER_URL }}
+  PACT_BROKER_USERNAME: ${{ secrets.PACT_BROKER_USERNAME }}
+  KNOWIT_EMAIL: ${{ secrets.KNOWIT_EMAIL }}
+jobs:
+  generate_typescript:
+    name: "Generate typescript types"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+      - name: Coursier cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/coursier
+          key: ${{ runner.OS }}-coursier-cache-${{ matrix.scala }}-${{ hashFiles('**/*.sbt') }}
+          restore-keys: |
+            ${{ runner.OS }}-coursier-cache-${{ matrix.scala }}-${{ hashFiles('**/*.sbt') }}-
+            ${{ runner.OS }}-coursier-cache-${{ matrix.scala }}-
+            ${{ runner.OS }}-coursier-cache-
+      - name: Ivy cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.ivy2/cache
+            ~/.sbt/boot/
+          key: ivy-cache-${{ hashFiles('~/.ivy2/cache/*') }}-${{ hashFiles('~/.sbt/boot/*') }}
```

Rydder du opp i cache tingene i alle disse, sånn som du gjorde i article-api (draft-api fikser jeg i min release pr)?

```suggestion
      - uses: coursier/cache-action@v6
```